test/jasmine/tests/geo_test.js

```diff
@@ -1353,46 +1353,6 @@ describe('Test geo interactions', function() {
         .then(done, done.fail);
     });
 
-    it('should clear hover label when cursor slips off subplot', function(done) {
```

Comment:
Why did these 2 tests get removed?

Reply:
If I recall (as mentioned in the commit message) there used to be a bug in d3.geo where a function returned `NaN` positions. And we guarded against it.

Reply:
But these tests are just using the high-level API, not d3 geo itself. If they started failing, that means something changed about the user-facing behavior. What changed? Are we comfortable with that change? I'd much prefer if we can adapt the tests to the new version so we can see the difference.

Reply:
Good question. Pointing to the fact that rebase somehow forgot to remove `geo/projections`. 
It is now fixed in https://github.com/plotly/plotly.js/pull/5112/commits/557ebde74e01c6df45c59083fc7c570a3e21ad70.

Reply:
This test should still be able to pass though. Perhaps the exact pixel positions need to change because there's some small change in how the projection is applied? If I try to manually follow what's going on in the test (I believe it's hovering on the point off the coast of Africa, then slowly mousing off the edge of the globe) the behavior looks identical before/after the change. So we should be able to bring the test back, but if the exact values need to change a little (`px = 390`, `py = 290` and `if(px < 402)` that's fine.

Reply:
Good call. Adjusted in https://github.com/plotly/plotly.js/pull/5112/commits/647b0dc5576e2b57c1b3425bb152a631a24d3e06.
In fact with previous positions the tests didn't pass locally. But now the test now passes both locally and on the CI.